### PR TITLE
Users Controller and Views

### PR DESCRIPTION
**Pull request by nawilliams**
Also adds Bootstrap Select

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.active
+    # TODO: should by default only display users that belong to
+    # the supervisor looking at this page
+    # permissions should also be locked to supervisors for this controller
+    # TODO: needs a way to display inactive users
+  end
+
+  def show
```
Comment (Anbranin) [inline app/controllers/users_controller.rb]: You don't need an empty controller action in Rails 5+

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  def edit
+    @user = User.find(params[:id])
+    @supervisors = User.active.where(is_supervisor: true)
+    @divisions = Division.all
+    @positions = Position.all
+  end
+
+  def update
+    if @user.update(user_params)
```
Comment (Anbranin) [inline app/controllers/users_controller.rb:38]: What's `@user` here?

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      flash[:success] = 'User Successfully Updated'
+      redirect_to users_path
+    else
+      flash[:warning] = 'User Could Not Be Updated'
+      redirect_to action: :edit
+    end
+  end
+
+  def destroy
+    @user = User.find(params[:id])
```
Comment (Anbranin) [inline app/controllers/users_controller.rb]: I don't see `@user` defined.

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    @user.update_attribute(:active, false)
+    flash[:success] = 'User Successfully Deleted'
+    redirect_to users_path
+    # TODO: if a user is a supervisor we should remove them as supervisor from their supervisees
```
Comment (Anbranin) [inline app/controllers/users_controller.rb]: a before_save action yas

**Review by Anbranin — APPROVED** [review]

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (nawilliams) [reply app/controllers/users_controller.rb]: Yeah, this was temporary as I hadn't created a show page and wasn't sure if we needed one. Removed for now.

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (nawilliams) [reply app/controllers/users_controller.rb:38]: Good catch, fixed

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (nawilliams) [reply app/controllers/users_controller.rb]: This is where it's defined 👍

Comment (nawilliams) [reply app/controllers/users_controller.rb]: Moved it to a before_action because that makes more sense.

**Pull request merged**